docs/configuration.md

```diff
@@ -80,7 +80,7 @@ Property
 **envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.protocol.cipher-suites**                                                    | A list of cipher suites to use                                                                                                                                                                 | list of "ECDHE-ECDSA-AES128-GCM-SHA256", "ECDHE-RSA-AES128-GCM-SHA256"
 **envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.mtls-enabled-tag**                                                          | Name of tag that is used to identify if mTLS should be enabled for cluster (note that `envoy-control.envoy.snapshot.egress.http2.enabled` should also be enabled)                              | mtls:enabled
 **envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.san-uri-format**                                                            | URI format for SAN field that will be matched in client/server validation. {service-name} will be replaced by service name from discovery source.                                              | spiffe://{service-name}
-**envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.san-uri-wildcard-regex**                                                    | URI format for SAN field for services that use wildcard clients. This means that every client that presents with a valid certificate and his client name matches this regex will be allowed.   | spiffe://.+
+**envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.service-name-wildcard-regex**                                               | Wildcard regex used put in san-uri-format to allow all authenticated clients to be allowed. If the values of this and san-uri-format are default the resulting regex is: spiffie://.+          | .+
```

Comment:
It doesn't describe where it is used in san-uri-format.

Maybe we could write it simpler and more generally:

> Regex to match service-names for "wildcard" client identifier. By default it will match all service names of length greater than zero (`.+`). It is used in place of `{service-name}` placeholder in san-uri-format.